RESPONSIBILITIES.md

```diff
@@ -38,7 +38,7 @@ Maintainers are active and visible members of the community, and have [maintain-
 
 ### Uphold Code of Conduct
 
-Model the behavior set forward by the [Code of Conduct](CODE_OF_CONDUCT.md) and raise any violations to other maintainers and admins.
+Model the behavior set forward by the [Code of Conduct](CODE_OF_CONDUCT.md) and raise any violations to other maintainers and [org admin](https://github.com/orgs/opensearch-project/teams/admin).
```

Comment:
Folks in the "admin" team aren't org admins. They are "members of the admin team". An org admin has some capabilities such as deleting the org, which none of the folks in the admin team are. Let's rename this to "members of the admin team".

Reply:
Thanks @dblock  updated, take a look